src/dotnet/App.Maui/Platforms/Android/AndroidWebViewClient.cs

```diff
@@ -10,58 +10,54 @@ public class AndroidWebViewClient(
     ILogger log
     ) : WebViewClient
 {
-    private ILogger? _log;
-
-    private ILogger Log => _log ??= StaticLog.Factory.CreateLogger(GetType());
+    private ILogger Log => log;
     private WebViewClient Original { get; } = original;
     private AndroidContentDownloader ContentDownloader { get; } = contentDownloader;
+    private bool HasDisconnected { get; set; }
```

Comment:
`has` is slightly confusing cause in variable naming context usually implies belonging. Maybe `is`? 🙂

Reply:
fixed